tools/clang/test/HLSLFileCheck/hlsl/semantics/legacy/obsolete_semantics_ps_3.hlsl

```diff
@@ -0,0 +1,8 @@
+// RUN: %dxc -E main -T ps_6_0 %s -Gec | FileCheck %s
+
+// CHECK: warning: Ignoring unsupported 'VFACE' in the target attribute string
+
+float4 main(float4 color : COLOR, float vface : VFACE) : SV_TARGET
```

Comment:
Regardless, if vface is a float, a validation error will be produced thanks to the change Vishal made I mentioned. So this must be uint.

Reply:
done